nutaudet/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(INCLUDE_DIRECTORIES
 ${BASE_INCLUDE_DIRECTORIES}
 ${CMAKE_SOURCE_DIR}/shipdata
 ${CMAKE_SOURCE_DIR}/nutaudet
+${ROOT_INCLUDE_DIR}
```

Comment:
Which headers is this needed for?

It would be better to be more explicit about which paths need to be included, as $ROOT_INCLUDE_DIR includes a lot of stuff.